log/json_logger_test.go

```diff
@@ -10,16 +10,67 @@ import (
 )
 
 func TestJSONLogger(t *testing.T) {
+	t.Parallel()
 	buf := &bytes.Buffer{}
 	logger := log.NewJSONLogger(buf)
 	if err := logger.Log("err", errors.New("err"), "m", map[string]int{"0": 0}, "a", []int{1, 2, 3}); err != nil {
 		t.Fatal(err)
 	}
 	if want, have := `{"a":[1,2,3],"err":"err","m":{"0":0}}`+"\n", buf.String(); want != have {
+		t.Errorf("\nwant %#v\nhave %#v", want, have)
+	}
+}
+
+func TestJSONLoggerMissingValue(t *testing.T) {
+	t.Parallel()
+	buf := &bytes.Buffer{}
+	logger := log.NewJSONLogger(buf)
+	if err := logger.Log("k"); err != nil {
+		t.Fatal(err)
+	}
+	if want, have := `{"k":"(MISSING)"}`+"\n", buf.String(); want != have {
+		t.Errorf("\nwant %#v\nhave %#v", want, have)
```

Comment:
Not necessary, but rather helpful when the test fails IMO. It creates vertical alignment that human pattern recognition deals with very efficiently, especially when the message is long and noisy as with JSON.

Without:
```
json_logger_test.go:32: want "{\"k\":\"(MISSING)\"}\n", have "{\"k\":\"(MISING)\"}\n"
```

With:
```
json_logger_test.go:32: 
	want "{\"k\":\"(MISSING)\"}\n"
	have "{\"k\":\"(MISING)\"}\n"
```